pxr/usdImaging/usdviewq/appController.py

```diff
@@ -2804,9 +2826,13 @@ def _getSaveFileName(self, caption, recommendedFilename):
         
         return saveName
 
+    def _getRecommendedFilenamePrefix(self):
+        return (self._parserData.usdFile.rsplit('.', 1)[0]
+                if self._parserData.usdFile
+                else 'new_file')
```

Comment:
Should this be something unique, like when we are creating temp files?  It's probably unlikely, but if a bunch of "file-less" usdview's are launched and you do live-editing and want to save out the overrides from more than one, if you go with the recommendation you'll stomp your results.  OTOH, you will see that the suggestion is the same for both, and have the ability to change it.  Just a thought to consider.

Reply:
... if you do go that route, given that there's multiple uses of this recommended name, and only one "empty/new stage" per usdview, Seems like we would want just one unique name per-process, right?

Reply:
Good point! The unique names usually look a bit ugly to people, so I wasn't sure about it :). I can replace this function with the following if people think this would be better.

```
def _getRecommendedFilenamePrefix(self):
    return (self._parserData.usdFile.rsplit('.', 1)[0]
            if self._parserData.usdFile
            else f'new_file_{os.getpid()}_{datetime.now().strftime("%Y%m%d_%H%M%S_%f")[:18]}')
```
The above function would make a unique name suggestion based on process ID and current datetime (millisecond accuracy), so almost unique everytime. But could be a bit jarring to look at. Open to suggestions :)

Here's an example of a potential suggestion from the above function:
`new_file_12345_20240927_154510_12`

We can potentially remove the `datetime` portion if we just need a unique name per process ID.

Thanks for the feedback, lmk if anything else stands out.

Reply:
Let's go ahead with the current naming, since file saving does ask for a confirmation if you try to override an existing file